qiskit/transpiler/passmanager.py

```diff
@@ -320,6 +322,14 @@ def passes(self) -> List[Dict[str, BasePass]]:
             else:
                 item["flow_controllers"] = {}
             ret.append(item)
+
+        def call(self):
+            warn("The `PassManager.passes()` method is deprecated as of 0.22.0, and "
+                  "will be removed no earlier than 3 months after that "
+                  "release date. You should use the `PassManager.passes` property instead.", DeprecationWarning, stacklevel=2)
+            return self
+
+        ret.__call__ = call
```

Comment:
I think for this to work in all situations you'll have to create a `list` subclass (or a `MutableSequence` type) and return that. Monkey patching a call definition on doesn't get preserved in a parallel context which is the source of the failures in the integration tests: https://github.com/Qiskit/qiskit-terra/runs/7046189043?check_suite_focus=true